src/main/java/com/phonebookservice/util/Messages.java

```diff
@@ -0,0 +1,13 @@
+package main.java.com.phonebookservice.util;
+
+public final class Messages {
+    private Messages() {
+    }
+
+    public static final String ERROR_DATABASE_NOT_FOUND //
+            = "ERROR_INVALID_RELATION";
+    public static final String ERROR_OBJECT_INVALID = "ERROR_OBJECT_INVALID";
+    public static final String ERROR_MANDATORY_FIELDS_MISSING //
```

Comment:
Message should be readable to the user. 
I think you confused between the follwoing 
1- ErrorCode  : Should be a number that identify the problem. e.g (lastName is null or firstName is null)
2- HttpCode : 400, 401, 403
3- ErrorMessage : message can be readbale from user